### PR TITLE
fix(build): standardize lint, improve some types

### LINKED ISSUE #383: Update `eslint` rules
I think we should update the ESLint rules. Rules rules are quite minimalistic and does not cover even the basic style issues (like trailing commas).

Some prefer using some style guides, I prefer using custom ESLint config and add rules that make sense to me, my code style and my coding/programming style. However, I am open to change any rule in the proposed ESLint config (see below). I don’t want to enforce any rule by myself, however, I want to enforce consistency.

Note that I compiled this config by myself and I am more of a TS newbie, the suggested config is not perfect and could be improved.

I commented on some stuff that you might want to consider, or when my preference differs from style currently used by romcal.

#### Some additional stuff to consider

- [ ] consider specifying `parserOptions.ecmaVersion` (should it be `es2015`?);
- [ ] consider using tabs for indentation (less characters to use, thus smaller source code size):
   - this would require changes in `.editorconfig` and `.prettierrc`;
- [ ] consider removing semicolons on EOLs;
- [ ] I prefer sorting JSON file properties, as well as TS/JS object properties (including TS type properties), thus could we sort `.eslintrc.json` too?
- [ ] let’s keep the trailing commas:
   - I prefer removing them, however, I understand that the diffs are more tidy when trailing commas are used;
- [ ] consider installing the following ESLint plugins (for rules and usage, see the proposed ESLint config below:

   ```bash
   npm -D i \
     # Note: This provides `plugin:import/recommended`.
     eslint-plugin-import \
     # Note: This provides `sort-keys-custom-order`.
     eslint-plugin-sort-keys-custom-order \
     # Note: This provides `eslint-plugin-tsdoc`.
     eslint-plugin-tsdoc \
     # Note: This provides `unicorn`.
     eslint-plugin-unicorn
   ```

#### Suggested configuration

<details>
   <summary><code>.eslintrc.json</code></summary>

```js
{
  "env": {
    "browser": false,
    "es6": true,
    "node": true
  },
  "extends": [
    "eslint:recommended",
    // Note: `eslint-plugin-import` plugin intends to support linting of ES2015+ (ES6+) import/export syntax, and prevent issues with misspelling of file paths and import names. All the goodness that the ES2015+ static module syntax intends to provide, marked up in your editor.
    "plugin:import/recommended",
    "prettier"
  ],
  "overrides": [
    {
      "files": [
        "*.js",
        "*.jsx"
      ],
      "rules": {
        // TODO: Why did I turn `indent` rule off for JS files? If needed look at the following config from old romcal ESLint config.
        "indent": [
          "error",
          // Note: I prefer `tab`. If we change this rule, update `tab` config in `.prettierrc` too.
          2,
          {
            "ArrayExpression": 1,
            "flatTernaryExpressions": false,
            "FunctionDeclaration": {"parameters": "first"},
            "ImportDeclaration": 1,
            "ObjectExpression": 1,
            "SwitchCase": 1,
            "VariableDeclarator": "first",
          }
        ],
        "no-extra-parens": ["error"]
      }
    },
    {
      "extends": [
        "plugin:@typescript-eslint/eslint-recommended",
        "plugin:@typescript-eslint/recommended"
      ],
      "files": [
        "*.ts",
        "*.tsx"
      ],
      "rules": {
        // Note: `@typescript-eslint/explicit-function-return-type` makes sure no explicit `return` is used in functions/methods that return `void`.
        "@typescript-eslint/explicit-function-return-type": "error",
        // Note: `@typescript-eslint/explicit-member-accessibility` requires to use explicit `public`, `protected` and `private` modifiers. With `accessibility` set to `no-public`, it does not requires explicit `public` modifier.
        "@typescript-eslint/explicit-member-accessibility": ["error", {"accessibility": "no-public"}],
        // Note: `@typescript-eslint/explicit-module-boundary-types` requires function/method explicit return types.
        "@typescript-eslint/explicit-module-boundary-types": "error",
        // Note: `@typescript-eslint/indent` is a bit buggy (see [typescript-eslint/typescript-eslint#1824](https://github.com/typescript-eslint/typescript-eslint/issues/1824), for my use cases, the following config works quite well (the fix is based on [this](https://github.com/typescript-eslint/typescript-eslint/issues/1824#issuecomment-957559729)).
        "@typescript-eslint/indent": [
          "error",
          // Note: I prefer `tab`. If we change this rule, update `tab` config in `.prettierrc` too.
          2,
          {
            "ArrayExpression": 1,
            "flatTernaryExpressions": false,
            "FunctionDeclaration": {"parameters": "first"},
            "ignoredNodes": [
              "FunctionExpression > .params[decorators.length > 0]",
              "FunctionExpression > .params > :matches(Decorator, :not(:first-child))",
              "ClassBody.body > PropertyDefinition[decorators.length > 0] > .key"
            ],
            "ImportDeclaration": 1,
            "ObjectExpression": 1,
            "SwitchCase": 1,
            "VariableDeclarator": "first"
          }
        ],
        // TODO: Was `@typescript-eslint/interface-name-prefix` removed from the plugin package?
        "@typescript-eslint/interface-name-prefix": "off",
        // TODO: We might want to set `@typescript-eslint/no-explicit-any` to `error`.
        "@typescript-eslint/no-explicit-any": "off",
        // Note: I prefer removing extra brackets, while others might want to keep them because of readability.
        "@typescript-eslint/no-extra-parens": ["error"],
        // Note: In my experience, `@typescript-eslint/no-floating-promises` causes some false-positives when using MikroORM (when I build a query, no `await` should be used; `await` needs to be added only when the query needs to be executed), however, it helped me a lot when I forgot to use `await`.
        "@typescript-eslint/no-floating-promises": ["error"],
        // Note: `@typescript-eslint/no-non-null-assertion` rule comes from current ESLint config, which we use because of `strictNullChecks` is set to `true` in `tsconfig.conf`.
        "@typescript-eslint/no-non-null-assertion": "off",
        // Note: In Nest.js, I use a lot parameter properties in class constructors, thus I simply disable this rule. However, ATM I am not sure if I had to disable this because of Nx (monorepo) ESLint rules or not. Maybe this is not necessary.
        "@typescript-eslint/parameter-properties": "off",
        // Note: `no-extra-parens` needs to be `off` when `@typescript-eslint/no-extra-parens` is
        "no-extra-parens": "off"
      }
    }
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": {
    // TODO: `ecmaVersion` could be changed.
    "ecmaVersion": 2015,
    "project": "tsconfig.base.json",
    "sourceType": "module"
  },
  "plugins": [
    "@typescript-eslint",
    "@typescript-eslint/eslint-plugin",
    "eslint-plugin-tsdoc",
    "jest",
    // Note: `sort-keys-custom-order` sorts object/type properties.
    // TODO: Note that `sort-keys-custom-order` is not perfect and it might require running ESLint multiple times in order to get a large object sorted (see [hugoattal/eslint-plugin-sort-keys-custom-order#2](https://github.com/hugoattal/eslint-plugin-sort-keys-custom-order/issues/2)).
    "sort-keys-custom-order",
    // Note: `unicorn` provides many (currently 106, see [here](https://github.com/sindresorhus/eslint-plugin-unicorn#rules)) useful rules, however, I don’t want to use `unicorn/recommended`, as I’d like to cherry-pick rules I want. I haven’t checked all the rules yet.
    "unicorn"
  ],
  "root": true,
  "rules": {
    // Note: I prefer `never` for `array-bracket-spacing`.
    "array-bracket-spacing": ["error", "always"],
    "arrow-body-style": ["error", "as-needed"],
    "arrow-parens": ["error", "as-needed"],
    // Note: I prefer `never` for `comma-dangle`. If we change this rule, update `trailingComma` config in `.prettierrc` too.
    "comma-dangle": ["error", "always-multiline"],
    // Note: I prefer using comma the way we do it in text in books: no space before comma, a space or newline after it.
    "comma-spacing": [
      "error",
      {
        "before": false,
        "after": true
      }
    ],
    // Note: I prefer `never` for `computed-property-spacing`.
    "computed-property-spacing": ["error", "always"],
    // Note: I prefer using curly brackets (for scope) consistently. I always use curly brackets in `if` and `else` statements.
    "curly": "error",
    // Note: `for-direction` simply checks if the loop can stop.
    "for-direction": "error",
    // Note: `import/named` verifies that all named imports are part of the set of named exports in the referenced module.
    "import/named": "error",
    // Note: `import/no-unresolved` ensures an imported module can be resolved to a module on the local filesystem, as defined by standard Node `require.resolve` behavior. I have disabled it, as it could not recognise `path` config in `tsconfig.json` and some other stuff (maybe `import/no-unresolved` should be configured somehow).
    "import/no-unresolved": "off",
    // Note: `import/order` makes sure that the order and groups of imports is consistent. Below is my custom config where I order imports (within a group) alphabetically (ascending, ignoring case) and I using four groups; between each group there needs to be at least one newline (`no-multiple-empty-lines` makes sure that there is one newline only).
    "import/order": [
      "error",
      {
        "alphabetize": {
          "caseInsensitive": true,
          "order": "asc"
        },
        "groups": [
          "builtin",
          "external",
          [
            "internal",
            "parent",
            "sibling",
            "index"
          ],
          [
            "object",
            "type"
          ]
        ],
        "newlines-between": "always"
      }
    ],
    "indent": "off",
    // Note: Make sure there is no space before a colon and one or more spaces (I’d prefer a single space) after a colon between property name and value in objects.
    "key-spacing": [
      "error",
      {
        "singleLine": {
          "afterColon": true,
          "beforeColon": false
        },
        "mode": "strict",
        "multiLine": {
          "afterColon": true,
          "beforeColon": false
        }
      }
    ],
    // Note: `keyword-spacing` enforces consistent spacing before and after keywords.
    "keyword-spacing": [
      "error",
      {
        "after": true,
        "before": true
      }
    ],
    // Note: I have set this rule `off`, however, it never hurts to make sure we use only `LF`/`\n`.
    "linebreak-style": ["error", "unix"],
    // TODO: One [this discussion](https://github.com/eslint/eslint/discussions/16142) has a solution, I’d like to enforce newlines before and after multi-line class members (of course, not at the beginning and at the end of the class).
    // "lines-between-class-members": ["error", "always"],
    // Note: We might want to remove this rule, as we use `prettier` in romcal. Also `prettier` uses `printWidth` variably, while `max-len` is used constantly (max `x` chars on a line, otherwise error out), which might cause som troubles.
    "max-len": [
      "error",
      {
        "code": 120,
        "ignoreComments": true,
        "ignoreRegExpLiterals": true,
        "ignoreStrings": true,
        "ignoreTemplateLiterals": true,
        "ignoreTrailingComments": true,
        "ignoreUrls": true,
        "tabWidth": 2
      }
    ],
    // Note: `new-cap` makes sure constructor names to begin with a capital letter.
    "new-cap": "error",
    // Note: `no-case-declarations` disallow lexical declarations in `case` clauses. I have turned this off in order to make my life easier, however, I’d prefer setting it to `error`.
    "no-case-declarations": "off",
    // Note: I prefer using case fallthrough when the actions (code) of multiple cases are same or nearly the same.
    "no-fallthrough": "off",
    // Note: `no-lonely-if` disallows if statements as the only statement in else blocks.
    "no-lonely-if": "error",
    // Note: `no-mixed-spaces-and-tabs` disallows mixed spaces and tabs for indentation.
    "no-mixed-spaces-and-tabs": "error",
    // Note: `no-new-wrappers` disallows `new` operators with the `String`, `Number`, and `Boolean` objects.
    "no-new-wrappers": "error",
    // Note: `no-multiple-empty-lines` disallows multiple empty lines.
    "no-multiple-empty-lines": [
      "error",
      {
        "max": 1,
        "maxBOF": 0,
        "maxEOF": 0
      }
    ],
    // Note: `no-tabs` disallows using literal tabs for other purposes than indentation.
    "no-tabs": [
      "error",
      {
        "allowIndentationTabs": true
      }
    ],
    // Note: `no-trailing-spaces` disallows trailing whitespaces on a line. This rule _might_ be redundant if we use `.editorconfig`, however, not all contributors might use it.
    "no-trailing-spaces": "error",
    // Note: `no-useless-concat` disallows unnecessary concatenation of literals or template literals.
    "no-useless-concat": "error",
    // Note: `no-var` inforces usage of `let` or `const` instead of `var`.
    "no-var": "error",
    // Note: `object-curly-spacing` enforces consistent spacing inside braces. I prefer setting it to `never` (i.e. write it as we would do that in a text of a book).
    "object-curly-spacing": ["error", "always"],
    // Note: `object-property-newline` enforces placing object properties on separate lines unless all properties are on the same line.
    "object-property-newline": [
      "error",
      {
        "allowAllPropertiesOnSameLine": true
      }
    ],
    // Note: `padded-blocks` requires or disallows padding within blocks. I prefer using no newlines after a block starts or before a block ends.
    "padded-blocks": ["error", "never"],
    // Note: `prefer-template` requires template literals instead of string concatenation.
    "prefer-template": "error",
    // Note: `quote-props` requires quotes around object literal property names. I prefer quoting only properties that are needed to be quoted, otherwise no quotes should be used.
    "quote-props": ["error", "as-needed"],
    // Note: `quotes` enforce the consistent use of either backticks, double, or single quotes.
    "quotes": [
      "error",
      "single",
      {
        "allowTemplateLiterals": true,
        "avoidEscape": true
      }
    ],
    // Note: `semi` requires or disallows semicolons instead of ASI. I prefer removing unnecessary semicolons. If we change this, update `semi` config in `.prettierrc`.
    "semi": ["error", "always"],
    // Note: `semi-spacing` enforces consistent spacing before and after semicolons.
    "semi-spacing": [
      "error",
      {
        "before": false,
        "after": true
      }
    ],
    // Note: `sort-imports` enforces sorted import declarations within modules.
    "sort-imports": [
      "error",
      {
        "allowSeparatedGroups": true,
        "ignoreCase": true,
        "ignoreDeclarationSort": true,
        "ignoreMemberSort": false,
        "memberSyntaxSortOrder": [
          "none",
          "all",
          "multiple",
          "single"
        ]
      }
    ],
    // Note: `sort-keys-custom-order/object-keys` sorts properties inside JS objects.
    "sort-keys-custom-order/object-keys": "error",
    // Note: `sort-keys-custom-order/type-keys` sorts properties inside TS types.
    "sort-keys-custom-order/type-keys": "error",
    // Note: `sort-vars` requires variables within the same declaration block to be sorted.
    "sort-vars": [
      "error",
      {
        "ignoreCase": true
      }
    ],
    // Note: `space-before-blocks` enforces consistent spacing before blocks.
    "space-before-blocks": "error",
    // Note: `spaced-comment` Enforce consistent spacing after the `//` or `/*` in a comment.
    "spaced-comment": ["error", "always"],
    // Note: `space-infix-ops` requires spacing around infix operators.
    "space-infix-ops": "error",
    // Note: `switch-colon-spacing` enforces spacing around colons of switch statements.
    "switch-colon-spacing": [
      "error",
      {
        "before": false,
        "after": true
      }
    ],
    // Note: `template-curly-spacing` requires or disallows spacing around embedded expressions of template strings. I prefer no spaces inside of brace brackets, thus I prefer use `never`.
    "template-curly-spacing": ["error", "never"],
    // Note: `tsdoc/syntax` validates TSDocs. I set it to `warn` as sometimes I miss some JSDoc features.
    "tsdoc/syntax": "warn",
    // Note: `unicorn/better-regex` improves regexes by making them shorter, consistent, and safer.
    "unicorn/better-regex": "error",
    // Note: `unicorn/catch-error-name` enforce a specific parameter name in catch clauses. I prefer using `e` in `catch`.
    "unicorn/catch-error-name": [
      "error",
      {
        "name": "e"
      }
    ],
    // Note: `unicorn/consistent-destructuring` requires destructured variables over properties.
    "unicorn/consistent-destructuring": "error",
    // Note: `unicorn/custom-error-definition` enforces correct `Error` subclassing.
    "unicorn/custom-error-definition": "error",
    // Note: `unicorn/empty-brace-spaces` enforces no spaces between braces. I prefer `error`.
    "unicorn/empty-brace-spaces": "off",
    // Note: `unicorn/error-message` enforces passing a `message` value when creating a built-in error.
    "unicorn/error-message": "error",
    // Note: `unicorn/prefer-module` prefers JavaScript modules (ESM) over CommonJS.
    "unicorn/prefer-module": "error",
    // Note: `unicorn/prefer-node-protocol` prefers using the `node:` protocol when importing Node.js builtin modules.
    "unicorn/prefer-node-protocol": "error"
  }
}
```

</details>

### PR DESCRIPTION
**Pull request by TobiTenno**
closes #383 

- update eslint to use airbnb (a very standard rule) for linting base
- lint more stuff by using overrides instead of not linting everything
- fix an (as far as i've seen) unreported bug around a type in a string
	- `PROPER_SOLEMNITY__TITLE_OR_FOUNDER_OR_PRIMARY_PATRON_OF_A_RELIGIOUS_ORG_8D`

### CODE REVIEW

**Comment by TobiTenno** [comment]
ooof, i forgot to make sure i ran tests locally. back to draft!

**Comment by TobiTenno** [comment]
@emagnier @tukusejssirs now that i've gotten the lint parts, i'm rather confused at this part, as i've not seen tmp built before build is run, so it's expecting some sort of tmp file to exist.

could one of you make it clearer for me?

File: scripts/build.ts [hunk 1]
```diff
@@ -96,26 +97,25 @@ log(chalk.bold(`\n  –– ${chalk.red('Romcal')} builder ––`));
...
-      `import { toPackageName } from "../../lib/utils/string";\n\n` +
-        `export const LOCALE_VAR_NAMES: string[] = ${JSON.stringify(localeNames)};\n\n` +
-        `export const LOCALE_IDS: string[] = LOCALE_VAR_NAMES.map(c => toPackageName(c));\n`,
-    ),
-    'utf-8',
+    await formatCode(`import { toPackageName } from "../../lib/utils/string";
+
+    export const LOCALE_VAR_NAMES: string[] = ${JSON.stringify(localeNames)};
+    export const LOCALE_IDS: string[] = LOCALE_VAR_NAMES.map(c => toPackageName(c));
+    `),
```
Comment (tukusejssirs) [inline scripts/build.ts]:
```suggestion
    await formatCode(
      `import { toPackageName } from "../../lib/utils/string";\n\n` +
        `export const LOCALE_VAR_NAMES: string[] = ${JSON.stringify(localeNames)};\n\n` +
        `export const LOCALE_IDS: string[] = LOCALE_VAR_NAMES.map(c => toPackageName(c));\n`
    ),
```

File: scripts/build.ts [hunk 2]
```diff
@@ -96,26 +97,25 @@ log(chalk.bold(`\n  –– ${chalk.red('Romcal')} builder ––`));
...
-        `export const CALENDAR_VAR_NAMES: string[] = ${JSON.stringify(calendarNames)};\n\n` +
-        `export const CALENDAR_PKG_NAMES: string[] = CALENDAR_VAR_NAMES` +
-        '.map(c => `@romcal/calendar.${toPackageName(c)}`);\n',
-    ),
-    'utf-8',
+    await formatCode(`import { toPackageName } from "../../lib/utils/string";
+
+      export const CALENDAR_VAR_NAMES: string[] = ${JSON.stringify(calendarNames)};
+      export const CALENDAR_PKG_NAMES: string[] = CALENDAR_VAR_NAMES.map(c => \`@romcal/calendar.\${toPackageName(c)}\`);
+    `),
```
Comment (tukusejssirs) [inline scripts/build.ts]:
```suggestion
    await formatCode(
      `import { toPackageName } from "../../lib/utils/string";\n\n` +
        `export const CALENDAR_VAR_NAMES: string[] = ${JSON.stringify(calendarNames)};\n\n` +
        `export const CALENDAR_PKG_NAMES: string[] = CALENDAR_VAR_NAMES` +
        '.map(c => `@romcal/calendar.${toPackageName(c)}`);\n',
    ),
```

**Review by tukusejssirs — CHANGES_REQUESTED** [review]
You have run the new ESlint config and it made some errors in the build script. The proposed two reverts sortes this out, although you still need to fix the ESlint errors. Until those are fixed, you need to run `pnpm clean && node -r ts-node/register scripts/build.ts` in order to circumwent running the `prebuild` script which checks the linting of the code, but fails.

OFC, you can still style the reverted code, you just need to make it works. :wink:

**Comment by emagnier** [comment]
Hello both, thank you @tukusejssirs and @TobiTenno for your contributions and thanks @TobiTenno for this PR! I'm indeed quite busy and have limited time available lately, but I'll take a look at your PR (and all the other recent PRs) asap 🙏

**Comment by TobiTenno** [comment]
> Hello both, thank you @tukusejssirs and @TobiTenno for your contributions and thanks @TobiTenno for this PR! I'm indeed quite busy and have limited time available lately, but I'll take a look at your PR (and all the other recent PRs) asap 🙏

I'd love to help out more, it'd be great if @tukusejssirs could tag-team PRs to get them through so we could help out and move things along faster.

we'd like to get all the verifications more stable, etc, and be able to update the public page so we don't have the kind of lag we're seeing now on fixes getting uploaded

**Comment by tukusejssirs** [comment]
> it'd be great if @tukusejssirs could tag-team PRs

@TobiTenno, what do you mean by _tag-team_ PRs? PRs tagged with `team` label?

**Comment by TobiTenno** [comment]
> > it'd be great if @tukusejssirs could tag-team PRs
> 
> @TobiTenno, what do you mean by _tag-team_ PRs? PRs tagged with `team` label?

@tukusejssirs sorry, it's an American colloquialism for pair-coding or working together to make sure we get through all of it

**Comment by TobiTenno** [comment]
@emagnier simply, it'd be great if we could have the ability to get a bit more through and done and have a wider write-enabled contributor base (like, even just @tukusejssirs) so that we can get fixes through and released

**Comment by tukusejssirs** [comment]
> @tukusejssirs sorry, it's an American colloquialism for pair-coding or working together to make sure we get through all of it

Thanks for the explanation! I’ve never heard it before. :wink:

> it'd be great if we could have the ability to get a bit more through and done and have a wider write-enabled contributor base (like, even just @tukusejssirs) so that we can get fixes through and released

Well, I have some rights (I can merge changes, though I usually let the maintainer [in this case Etienne] to do it). I don’t consider myself worthy of being an actual maintainer (due to my inexperience and lack of time I can devote to romcal). That said, I don’t have full permissions on the repo (I am only a [member of romcal org](https://github.com/orgs/romcal/people)). I think that might be worthy to have full access to romcal repo (i.e. to become an owner) which I’d use rarely. WDYT, @emagnier?

Also, it seems to me that @TobiTenno has some time to devote to romcal and he seems to me like he has enough experience to maintain romcal, therefore, I suggest to let him become a (co-) maintainer. WDYT, @emagnier?

**Review by tukusejssirs — APPROVED** [review]

**Comment by tukusejssirs** [comment]
>     * fix an (as far as i've seen) unreported bug around a type in a string
>       
>       * `PROPER_SOLEMNITY__TITLE_OR_FOUNDER_OR_PRIMARY_PATRON_OF_A_RELIGIOUS_ORG_8D`

This change is a bit outside of the scope of the #383, consider moving it to a separate PR. However, even if not moved to a separate PR, I’ll merge this branch tomorrow in the evening CET (~5-7pm UTC).

**Comment by TobiTenno** [comment]
> > ```
> > * fix an (as far as i've seen) unreported bug around a type in a string
> >   
> >   * `PROPER_SOLEMNITY__TITLE_OR_FOUNDER_OR_PRIMARY_PATRON_OF_A_RELIGIOUS_ORG_8D`
> > ```
> 
> This change is a bit outside of the scope of the #383, consider moving it to a separate PR. However, even if not moved to a separate PR, I’ll merge this branch tomorrow in the evening CET (~5-7pm UTC).

@tukusejssirs I'm happy to move it to help clarity and observability of the bug

**Comment by TobiTenno** [comment]
> > > ```
> > > * fix an (as far as i've seen) unreported bug around a type in a string
> > >   
> > >   * `PROPER_SOLEMNITY__TITLE_OR_FOUNDER_OR_PRIMARY_PATRON_OF_A_RELIGIOUS_ORG_8D`
> > > ```
> > 
> > 
> > This change is a bit outside of the scope of the #383, consider moving it to a separate PR. However, even if not moved to a separate PR, I’ll merge this branch tomorrow in the evening CET (~5-7pm UTC).
> 
> @tukusejssirs I'm happy to move it to help clarity and observability of the bug

sorry, I say that, but it's actually caught _because_ of the changes w/ #383, essentially it's the bug that got caught by linting/typing better, and removing it would cause a lint error that would make this fail CI

**Comment by tukusejssirs** [comment]
> sorry, I say that, but it's actually caught because of the changes w/ https://github.com/romcal/romcal/issues/383, essentially it's the bug that got caught by linting/typing better, and removing it would cause a lint error that would make this fail CI

Okay, then it makes sense to keep it in this PR. :pray:

**Pull request merged**